lib/smart_answer_flows/student-finance-calculator/questions/do_any_of_the_following_apply_uk_full_time_students_only.govspeak.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :title do %>
-  Do any of the following apply? (you might get extra funding.)
+  Do any of the following apply? (You might get extra funding.)
```

Comment:
On this page, there's an `eg` on line 9 in the `"low-income"` section which doesn't match the styleguide. The spreadsheet didn't say whether this should be changed or not.

Reply:
Change any eg you find to "for example" please.

Reply:
Thanks. Done!